src/core/client/stream/tabs/Comments/Stream/FeaturedCommentTooltip.tsx

```diff
@@ -18,20 +32,18 @@ export const FeaturedCommentTooltip: FunctionComponent<Props> = props => {
           <span>How is a comment featured?</span>
         </Localized>
       }
-      body={
-        <Localized id="comments-featuredCommentTooltip-handSelectedComments">
-          <span>Comments are hand selected by our team as worth reading.</span>
-        </Localized>
-      }
-      button={({ toggleVisibility, ref }) => (
+      body={<FeaturedCommentTooltipContent />}
+      button={({ toggleVisibility, ref, visible }) => (
         <Localized
           id="comments-featuredCommentTooltip-toggleButton"
           attrs={{ "aria-label": true }}
         >
           <TooltipButton
             active={props.active}
             aria-label="Toggle featured comments tooltip"
-            toggleVisibility={toggleVisibility}
+            toggleVisibility={() => {
```

Comment:
what's this for?

Reply:
It looks like a testing artifact (should be revised).